codespell_lib/data/dictionary.txt

```diff
@@ -16355,6 +16355,8 @@ subsytem->subsystem
 subsytems->subsystems
 subtabels->subtables
 subtances->substances
+subtarger->subtarget
```

Comment:
Perhaps also add the hyphenated variants of this and the plural?

Reply:
2982c26